dashboard/config/routes.rb

```diff
@@ -979,7 +979,7 @@
         post 'users/has_seen_ai_assessments_announcement', to: 'users#post_has_seen_ai_assessments_announcement'
         post 'users/disable_lti_roster_sync', to: 'users#post_disable_lti_roster_sync'
         post 'users/:user_id/ai_tutor_access', to: 'users#update_ai_tutor_access'
-        post 'users/:user_id/has_completed_ai_differentiation_welcome', to: 'users#post_has_completed_ai_differentiation_welcome'
+        post 'users/has_completed_ai_differentiation_welcome', to: 'users#post_has_completed_ai_differentiation_welcome'
```

Comment:
This is the part where I don't have enough context to thoroughly review - I assume there's some code in the `has_completed_ai_differentiation_welcome` controller action that is looking at `current_user` now rather than using a user_id?

Reply:
This was just me copy-pasting from the line above something that we don't need. There's no reason to include the user_id in the route and it just makes writing it a bit more confusing